kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue

```diff
@@ -507,7 +507,12 @@
       },
     },
     created() {
+      const welcomeDimissalKey = 'DEVICE_WELCOME_MODAL_DISMISSED';
```

Comment:
I'd suggest moving this constant into a common file, maybe in kolibri-common (for reuse across device, learn, and setup wizard plugins).

Reply:
It _is_ called **DEVICE**\_WELCOME_MODAL_DISMISSED not COMMON_... though 🤣

Reply:
Now that I've actually gone and tested the fix I realize the silliness of my previous comment 😬  😆